tools/fpgabist/bist_common.py

```diff
@@ -78,6 +80,13 @@ def get_mode_from_path(gbs_path):
     return None
 
 
+def is_command_in_path(cmd):
```

Comment:
I don't know if it makes sense to use it but `distutils.spawn`package has a function called `find_executable` that you may be able to use. Taken from the Python help - `help('distutils.spawn')`:
```
    find_executable(executable, path=None)
        Tries to find 'executable' in the directories listed in 'path'.
        
        A string listing directories separated by 'os.pathsep'; defaults to
        os.environ['PATH'].  Returns the complete filename or None if not found.
```